.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'monthly'
+    open-pull-requests-limit: 10
+    commit-message:
+      prefix: 'chore'
+    groups:
+      # Any updates not caught by the group config will get individual PRs
+      gha-low-risk:
+        update-types:
+          - 'minor'
+          - 'patch'
+
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'monthly'
+    open-pull-requests-limit: 10
+    commit-message:
+      prefix: 'chore'
+    ignore:
+      # Prevent updates to ESM-only versions
+      - dependency-name: 'chalk'
+        versions: ['>=5.0.0']
+      - dependency-name: 'esbuild'
+        versions: ['>=0.11.0']
+      - dependency-name: 'execa'
+        versions: ['>=6.0.0']
+      - dependency-name: 'globby'
```

Comment:
Can we replace this instead of needing to pin it? It isn't used in many spots and there are several alternatives that I expect would be viable